app/src/test/java/invoice_automation/module/QuickBooksModuleTest.java

```diff
@@ -75,6 +80,11 @@ private void setupMethodTests() throws Exception {
         quickBooksModule = new QuickBooksModule(ACCESS_TOKEN, REALM_ID, false);
     }
 
+    private void setupInvoiceTests() {
+        invoice = new Invoice();
+        invoice.setBillEmail(billEmail);
```

Comment:
Let's move this line to each unit tests which actually requires the billing email to be set, since future invoice tests don't necessarily need `billEmail` to be set